arctic/store/_pandas_ndarray_store.py

```diff
@@ -186,7 +186,7 @@ def _index_range(self, version, symbol, date_range=None, **kwargs):
                     return -1, -1
                 idxstart = min(np.searchsorted(dts, start), len(dts) - 1)
                 idxend = min(np.searchsorted(dts, end), len(dts) - 1)
-                return index['index'][idxstart], index['index'][idxend] + 1
+                return int(index['index'][idxstart]), int(index['index'][idxend] + 1)
```

Comment:
In the most recent numpy they broke the link between numpy ints and python ints (they no longer subclass the python type). This confuses the mongodb driver..